_events/baectf.md

```diff
@@ -0,0 +1,92 @@
+---
+name: "Capture the Flag with BAE Systems"
+is-public: true
+is-over: false
+
+start: "2019-01-19T10:00:00+00:00"
+end: "2019-01-19T19:16:00+00:00"
+
+short-description: ""
+full-description: "Local Hack Day is a global hackathon uniting thousands of students. Participants across 100+ distributed campuses are able to connect with their local communities and other hackers around the world through digital workshops & mini-events, social media, and a global livestream."
```

Comment:
Did you mean to leave the local hack day description? Also, this needs a short description if possible :)